assets/css/app/components/_resource_usage.scss

```diff
@@ -21,64 +21,82 @@
   }
 
   // Section colors
-  .memory-usage-section-atom {
+  .resource-usage-solid-1 {
```

Comment:
Instead of using numbers, let's use colors please. If necessary, rename $color-data-1 to their approximate colors too.

Reply:
Im thinking about extracting the colors to a separate file and just call them solid-color and gradient-color 
then it won't be duplicated - and you really want the same green in every place green is used
so basically instead `.memory-usage-section-atom` having `.solid-green`

Reply:
I think that's a solid approach too. Basically utility classes like bg-green, bg-red and so on. @archdragon wdyt?